ui/Dockerfile-production

```diff
@@ -0,0 +1,9 @@
+FROM node:20
+
+COPY . /ui
+WORKDIR /ui
+
+RUN npm install
+
+CMD [ "npm", "run", "build"] 
```

Comment:
So when I switch over to using the `build` option it seems to break my application and make it unusable. Do the options `build` and `dev` that get used in this docker file correspond to this -> https://github.com/brainlife/ezbids/blob/8ddb0a0291ac5a9d36b3ac175602245c35843383/ui/package.json#L4

Really not sure what exactly is all going on here.

Reply:
What further confuses me is what effect this vite config has, see -> https://github.com/brainlife/ezbids/blob/8ddb0a0291ac5a9d36b3ac175602245c35843383/ui/vite.config.ts#L8-L20

Is this at all related to the page refreshing?